webapp/tests/forms/steps/lotse/test_personal_data_steps.py

```diff
@@ -573,3 +577,79 @@ def test_if_required_value_is_not_give_validation_should_be_failure(self, new_te
                 StepPersonBHasDisability.name, True, ImmutableMultiDict(data))
             form = step.render_info.form
             assert form.validate() is False
+
+class TestPauschBetragPersonAValidation:
```

Comment:
I think some tests for the `get_pauschbetrag` and `get_value_representation` methods are missing

Reply:
I think this is still the case?